DesktopEdge/Views/ItemRenderers/MenuEditItem.xaml

```diff
@@ -5,15 +5,15 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
              xmlns:local="clr-namespace:ZitiDesktopEdge"
              mc:Ignorable="d" MaxWidth="360"
-             d:DesignHeight="31" Height="31" HorizontalAlignment="Stretch" d:DesignWidth="350" Cursor="Hand">
+             d:DesignHeight="31" Height="31" HorizontalAlignment="Stretch" d:DesignWidth="350">
```

Comment:
Removal of the screen's assignment to have the hand cursor helped to clarify what could/not be clicked within the screen.